database/mssql/scripts/dbscripts/PSP_PIMS_S24_00/Test Data/08_CONTACTS_ORGANIZATION_TEST_DATA.sql

```diff
@@ -0,0 +1,27 @@
+-- Add more fields to seed data for read view of contact organization
```

Comment:
@dfilteau this file is an exact duplicate of 07_CONTACTS_ORGANIZATION_TEST_DATA.sql why was it added?
it seems like a large number of additional duplicates are being added in this PR as well. Why are changes to test data being made as part of this PR?